pkg/test-infra/tidb/template.go

```diff
@@ -74,7 +74,8 @@ then
 fi
 echo "start tidb-server ..."
 echo "/tidb-server ${ARGS}"
-exec /tidb-server ${ARGS}
+/tidb-server ${ARGS} 2>&1 | tee -a /var/log/tidblog/tidb.log
+#                               ^^: append mode
```

Comment:
How about set log-file to `/var/log/tidblog/tidb.log` in config file?

Pros: 
* It will auto rotate log
* configurable
* daily run case are controlled by admin, we can ensure that these cases use correct config

Reply:
That's troubling to me, I need such a tool to help me:
1. run the tikv-server in the background
2. log tail tikv.log to stdout
3. exit with the exit-code when tikv-server exits

Reply:
resolve in 448bc24